arm-resources/resources/2016-02-01/swagger/resources.json

```diff
@@ -1877,6 +1877,22 @@
         "properties": {
           "type": "object",
           "description": "Gets or sets the resource properties."
+        },
+        "kind": {
```

Comment:
Some knowledge Transfer: Specifying an enum in swagger is a little different then what it means for the server. On the wire it will be a type that is defined in the swagger. enum is just a constraint which describes the set of allowed values. To that AutoRest adds an extension named "x-ms-enum" which will generate an Enum in the client if modelAsString is false or is not specified. If modelAsString is true then AutoRest will generate a static class with constant properties. However, it will be serialized/deserialized as the underlying type specified in swagger.

So specifying an enum is a value add for customers. It guides them about the possible values rather than they trying to find it by trial and error. Thus enum should be added in the spec wherever possible.